inputfiles/overridingTypes.jsonc

```diff
@@ -1241,14 +1241,16 @@
                             "name": "append",
                             "flavor": "Web",
                             "overrideSignatures": [
-                                "append(name: string, value: string | Blob, fileName?: string): void"
+                                "append(name: string, value: string): void",
+                                "append(name: string, value: Blob, fileName?: string): void"
```

Comment:
`additionalSignatures` is probably better here. The original signatures were already looked like this and the overriding signature was intended to support passing argument with type `string | Blob`.

So please do this instead:

```json
"additionalSignatures": [
    "append(name: string, value: string | Blob): void;"
]
```

Same for `set()`. It will end up with three signatures and that should be fine.